serverless/develop/serverless-using-brokers.adoc

```diff
@@ -1,30 +1,30 @@
 :_content-type: ASSEMBLY
 [id="serverless-using-brokers"]
-= Brokers
+= Creating brokers
 include::_attributes/common-attributes.adoc[]
 :context: serverless-using-brokers
 
 toc::[]
 
-include::snippets/serverless-brokers-intro.adoc[]
+Knative provides a default, channel-based broker implementation. This channel-based broker can be used for development and testing purposes, but does not provide adequate event delivery guarantees for production environments.
 
-include::modules/serverless-broker-types.adoc[leveloffset=+1]
+If your {ServerlessProductName} deployment is not configured to use Kafka broker as the default broker type, the channel-based broker is created when you use the default settings in the following procedures.
 
-[id="serverless-using-brokers-creating-brokers"]
-== Creating a broker that uses default settings
+If a cluster administrator has configured your {ServerlessProductName} deployment to use Kafka as the default broker type, creating a broker by using the default settings creates a Kafka-based broker.
```

Comment:
Am I correct that these two paragraphs present opposite sides of a coin? If so, can we have the second paragraph first? I always like to have the positive item first.

Reply:
Good suggestion, thanks!

Reply:
Hi, @abrennan89. It looks like Lines 13 and 27 have the same content. Is that as intended (or maybe it's an editing illusion)?

Reply:
It's similar content yes but since there's so much other content (procedures) between the two, I think it's good to leave this in as an explanation of the subheading. I will be doing some more additional cleanup re structure of broker and kafka docs in other PRs most likely, so I'll revisit this then.